src/display/text_layer.js

```diff
@@ -226,7 +530,61 @@ var renderTextLayer = (function renderTextLayerClosure() {
           self._renderTimer = null;
         }, timeout);
       }
-    }
+    },
```

Comment:
Nit: I know that we haven't always been consistent, but let's put a newline here between the functions to make the code a bit easier to read.